### ISSUE TITLE
FAQ: Error when communicating with Google API(10) does not address current version

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-website#717]
## Where to find the issue
**[Google/Android]: Error when communicating with Google API(10)**
https://www.coronawarn.app/en/faq/#10

**[Google/Android]: URSACHE 3 – Etwas ist schief gelaufen. Fehler bei Kommunikation mit Google API(10)**
https://www.coronawarn.app/de/faq/#10

## Describe the issue
The FAQ says "This error is resolved by version 1.5 of the Exposure Notification System." which does not provide any guidance for a user experiencing the issue when running a version 1.5 or higher.

## Suggested change
Provide guidance to users who are experiencing the error
`ApiException: 10: Unable to validate key file signature: no public key found for package de.rki.coronawarnapp`
when the version of ENS is 1.5 or higher.

## Additional information
Issue "Google API Exception 10: Unable validate key file signature" https://github.com/corona-warn-app/cwa-app-android/issues/1962 is a report from a user with CWA 1.9.1 and ENS Version 18204713000 (which corresponds to Version 1.8, from week 47 of year 2020). This version is higher than Version 1.5 which the FAQ article states resolves the error.

**Comment by dsarkar** [comment]
@MikeMcC399 

Suggestion: Add the following to the FAQ (EN)

"
UPDATE 14.01.2020
Should you experience this issue, we would appreciate receiving your report here: https://github.com/corona-warn-app/cwa-app-android/issues/1962 
"
... then the remainder of the original FAQ entry.


Thanks!

**Comment by MikeMcC399** [comment]
@dsarkar 
Good suggestion.

In fact I recently saw this error myself after doing a factory reset on the device. It seemed somehow that Google Play had not got installed properly, so I reset the device again and I never saw the error again, even though I did several factory resets due to the general Google error regarding no updates. I didn't report it in corona-warn-app/cwa-app-android#1962 because I could not reproduce it. It did show though that the error can still occur with CWA 1.10.1, so it is good to update the FAQ.
